data/presets/man_made/flare.json

```diff
@@ -0,0 +1,25 @@
+{
+    "icon": "temaki-chimney",
+    "fields": [
+        "height"
+    ],
+    "moreFields": [
+        "ref",
+        "visibility"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
```

Comment:
What would the vertex case be?

Reply:
perhaps it could be connected to a pipeline, since the gas must come from somewhere? 

I'm not sure if it's commonly mapped as a vertex, but I didn't want to limit the preset given that I just opened #1233 😄

Happy to remove it if you think this is uncommon

Reply:
Looked at 5-10 examples and found one connected to a line, so lets keep it https://www.openstreetmap.org/node/5492815582